### PR TITLE
NOPS-943 update HEAD branch to main

### PR DESCRIPTION
**Pull request by jenniferemshepherd**

### CODE REVIEW

File: origami.json [hunk 1]
```diff
@@ -3,7 +3,7 @@
 	"origamiType": "component",
 	"supportStatus": "active",
 	"brands" : [
-		"master"
+		"main"
```
Comment (JakeChampion) [inline origami.json]: This isn't a branch name unfortunately, it is a brand name -- the brand name "master" is being renamed at some point but that work hasn't been finished just yet

```suggestion
		"master"
```

Comment (notlee) [reply origami.json]: 👍 [Here's the proposal](https://github.com/Financial-Times/origami/pull/65). I think the Brand team decided on renaming the master brand to core, I'll make a note to follow up with them to see if they have gone ahead with that

Comment (jenniferemshepherd) [reply origami.json]: so renaming this will fall under your migration work in future?

Comment (JakeChampion) [reply origami.json]: Yes 👍

**Review by notlee — APPROVED** [review]

**Comment by origamiserviceuser** [comment]
:tada: This PR is included in version v7.0.1 :tada:

**Pull request merged**